src/cpp/src/speech_recognition/paraformer_impl.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "paraformer_impl.hpp"
+
+namespace ov {
+namespace genai {
+
+ParaformerImpl::ParaformerImpl(const std::filesystem::path& model_dir,
```

Comment:
how do you get IRs for paraformer models? Do exporting and inference work in optimum-intel? If no, then we must add it for the full consistent user experience.